### ISSUE TITLE
Building a db file on a subset of the MIMIC-III data

### ISSUE THREAD

**Issue by pshuwei** [MIT-LCP/mimic-code#1579]
### Prerequisites

* [X ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Description of the issue, including:
* what you have tried
I have successfully managed to run and build a mimic db file through the shell program for sqlite.

I am just curious if these codes can be run to build a db file on a subset of the data.

Thanks!

**Comment by alistairewj** [comment]
Yes for sure! You can run the same code using the demo dataset: https://physionet.org/content/mimiciii/

That would give you a 100 patient subset.

**Comment by pshuwei** [comment]
Hi thanks for your response,

Does this MIMIC demo dataset contain the same amount of information for 100 patients, or is it simply a condensed version?

Also what if I wanted to increase from 100 to 200 patients? How would I go about that?
